sorting/sorting.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println(`The program sorts three numbers.
+	Enter the numbers:`)
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	if a > b {
```

Comment:
What if we had 5 or even 10 numbers here? Can you try to come up with a solution that scales better, and thus, requires less code?

_Hint: You can swap variable values._